README.markdown

```diff
@@ -194,19 +194,13 @@ Follow the instructions below if you are updating from Symphony 2.0 (not from Gi
 
 Symphony uses [Grunt](http://gruntjs.com/) to build concatenated and minified assets:
 
-1. Install `grunt-cli` globally ([see the Grunt documentation](http://gruntjs.com/getting-started)):
-
-		npm install -g grunt-cli
-
-2. Install all dependencies from the repository’s root:
+1. Install all dependencies from the repository’s root:
 
 		npm install
 
-3. Run the `watch` task:
-
-		grunt watch
```

Comment:
If I have to “build concatenated and minified assets” for whatever reason, the most straight forward way is to just run grunt. Watch is only useful while actively working on the assets during development. I guess we could/should actually mention and explain both, but the watch task is not the default or most convenient way to trigger a single build.